### PR TITLE
Update "How to delete your Arduino account" to have more information on account deletion and republish the article [HC-1222]

### PR DESCRIPTION
**Pull request by seaxwi**
Improves **How to delete your Arduino account** and republishes the article.

### CODE REVIEW

File: content/About Arduino/My Arduino Account/How-to-delete-your-Arduino-account.md [hunk 1]
```diff
@@ -36,4 +72,24 @@ You can also delete your account from the Science Journal app:
 
 4. Open the email associated with your Arduino account and look for an email from noreply<!-- nolink-->@arduino.cc.
```
Comment (Renat0Ribeir0) [inline content/About Arduino/My Arduino Account/How-to-delete-your-Arduino-account.md:73]:
```suggestion
4. Open the email associated with your Arduino account and look for an email from **noreply<!-- nolink-->@arduino.cc**.
```
Emphasis to keep consistency with line 59.

File: content/About Arduino/My Arduino Account/How-to-delete-your-Arduino-account.md [hunk 2]
```diff
@@ -1,32 +1,68 @@
 ---
 title: "How to delete your Arduino account"
 id: 360017090920
-draft: true
+draft: false
 ---
 
-Learn about Arduino account deletion.
+You can delete your Arduino account at any time. Learn what you should know before deleting your Arduino account, and how to requesst account deletion.
```
Comment (Renat0Ribeir0) [inline content/About Arduino/My Arduino Account/How-to-delete-your-Arduino-account.md:7]:
```suggestion
You can delete your Arduino account at any time. Learn what you should know before deleting your Arduino account, and how to request account deletion.
```

**Review by Renat0Ribeir0 — APPROVED** [review]
Left two suggestions, but LGMT otherwise 👍

**Comment by stevisco** [comment]
when we describe the request for anonymization for forum, in my opinion the last step should be more clear "Contact us to request account anonymization" - not sure if it's clear that the "delete account" step will NOT perform anonymization automatically and that they have to use the contact us form to request it. We can also explain that we don't do this automatically to allow them to find their posts and remove PIIs, which otherwise becomes impossible if we anonymize automatically...

**Pull request merged**